R/client.R

```diff
@@ -33,32 +33,46 @@ sync <- function(since = NULL) {
 #'
 #' @param room_id The room whose messages will be received.
 #' @param from The start token within the history.
+#' @param dir Direction to receive messages in. "b" for backwards in time, "f"
+#'   for forwards in time.
+#' @param to The end token to stop receiving messages at.
 #'
 #' @return A list object from the Matrix API.
 #'
 #' @export
-get_messages <- function(room_id, from) {
+get_messages <- function(room_id, from, dir = "b", to = NULL) {
```

Comment:
if `to` can be used, should `from` be optional now?

Reply:
According to the spec, `from` is required by the API. Which probably makes sense, because messages have to start somewhere.

Reply:
Fair enough